### ISSUE TITLE
It seems that the model is tested on training dataset for MINIST dataset.

### ISSUE THREAD

**Issue by xjgaocs** [facebookresearch/improved_vrnn#1]
Both train_dataset and val_dataset set a true for the MovingMNIST class (at line 26 of datasets.py).

**Comment by lluiscastrejonsubira** [comment]
Fixed by 196d00523d53c1956284a09c025c3a36d4710f63.

Thanks for catching it, in our internal code we tried fixing the validation set across models (the training dataset is generated randomly) so that the comparison was fairer, and this code wasn't properly modified for the public release, hence the mistake.
